examples/operator/operator.yaml

```diff
@@ -6155,19 +6880,3 @@ spec:
               memory: 128Mi
       priorityClassName: vitess-operator-control-plane
       serviceAccountName: vitess-operator
----
-apiVersion: scheduling.k8s.io/v1
-description: The vitess-operator control plane.
-globalDefault: false
-kind: PriorityClass
-metadata:
-  name: vitess-operator-control-plane
-value: 5000
----
-apiVersion: scheduling.k8s.io/v1
-description: Vitess components (vttablet, vtgate, vtctld, etcd)
-globalDefault: false
-kind: PriorityClass
-metadata:
-  name: vitess
-value: 1000
```

Comment:
I don't think we want to remove this.

Reply:
We are not removing this block, the tool moved it a bit earlier in the file. I have used the raw output of `kustomize` which I think is what we should do from now on to avoid any conflict or unnecessary changes in the future.

See https://github.com/planetscale/vitess-operator/pull/553#discussion_r1606831787